src/common/modules/navigations/lnb/modules/LNBMenuItem.vue

```diff
@@ -73,22 +73,17 @@ export default {
         PDivider,
     },
     props: {
-        hasTopTitle: {
-            type: Boolean,
-            default: false,
-        },
         menuData: {
             type: Array as () => LNBItemList,
             default: () => [],
         },
         currentRoute: {
-            type: String,
-            default: '',
+            type: Location,
+            default: undefined,
```

Comment:
엇 이게 동작하나요?? 혹시 워닝 안뜨는지요?
```suggestion
            type: Object as PropType<Location|undefined>,
            default: undefined,
```
음 이렇게 작성하는게 더 좋을 것 같습니다!